src/Module/Api/Mastodon/Search.php

```diff
@@ -196,6 +196,11 @@ private function searchHashtags(string $q, bool $exclude_unreviewed, int $limit,
 		$tags = DBA::select('tag', ['name'], $condition, $params);
 
 		$hashtags = [];
+
+		if (!is_iterable($tags)) {
+			return $hashtags;
+		}
```

Comment:
Please use `DBA::selectToArray` instead.
```suggestion
		$tags = DBA::selectToArray('tag', ['name'], $condition, $params);

		$hashtags = [];
```

Reply:
Nice, thank you. 👍